packages/core/package.json

```diff
@@ -70,7 +70,9 @@
     "jaro-winkler": "^0.2.8",
     "json-stringify-safe": "^5.0.1",
     "lodash": "^4.17.21",
+    "pg": "^8.11.3",
     "playwright": "^1.39.0",
+    "sequelize": "^6.37.1",
```

Comment:
Do we need both?

Reply:
Yea i only added the sequelize but then it failed because it also needed pg